### PR TITLE
Application for Verible

### PR DESCRIPTION
**Pull request by mgielda**
Signed-off-by: Tim 'mithro' Ansell <tansell@google.com>

### CODE REVIEW

File: projects/graduated/verible.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * GitHub handle: @hzeller
+ * Role within the project: Lead Developer
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers: https://github.com/google/verible/graphs/contributors
+* Brief description of release methodology and mechanics: Releases are automated and published on every commit.
+* Link to draft mission statement:
```
[the remark this answers is not in the document]
Comment (mithro) [reply projects/graduated/verible.md]: Done.

File: projects/graduated/verible.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* Link to logo in .svg format: None.
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance: Confirmed
+* Confirmation that the project will adopt the [CHIPS Alliance IP policy](https://technical-charter.chipsalliance.org) upon acceptance: Confirmed
+* Confirmation that the project will add CHIPS Alliance header or footer text and links to its websites upon acceptance: Confirmed
+* Confirmation that the project will transfer any registered trademarks and domain names to the Linux Foundation upon acceptance:
+* Link to documented process for reporting security vulnerabilities:
```
[the remark this answers is not in the document]
Comment (mithro) [reply projects/graduated/verible.md]: Done.

File: projects/graduated/verible.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+### Graduation application
+
+* Total number of active committers: https://github.com/google/verible/graphs/contributors
+* Brief description of release methodology and mechanics: Releases are automated and published on every commit.
+* Link to draft mission statement:
+* Link to logo in .svg format: None.
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance: Confirmed
+* Confirmation that the project will adopt the [CHIPS Alliance IP policy](https://technical-charter.chipsalliance.org) upon acceptance: Confirmed
+* Confirmation that the project will add CHIPS Alliance header or footer text and links to its websites upon acceptance: Confirmed
+* Confirmation that the project will transfer any registered trademarks and domain names to the Linux Foundation upon acceptance:
```
[the remark this answers is not in the document]
Comment (mithro) [reply projects/graduated/verible.md]: Done.

File: projects/graduated/verible.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+### Sandbox application
+
+* Project name: Verible
+* Project repo(s): https://github.com/google/verible
+* Brief summary of the project: The Verible project’s main mission is to parse SystemVerilog (IEEE 1800-2017) (as standardized in the [SV-LRM]) for a wide variety of applications, including developer tools.
+* Project's open source license: Apache 2.0
+* Link to issue tracker: https://github.com/google/verible/issues
+* Link to website: https://google.github.io/verible/
+* Links to social media accounts: None.
+* Who uses this project, and at what scale:
```
[the remark this answers is not in the document]
Comment (mithro) [reply projects/graduated/verible.md]: Done.

File: projects/graduated/verible.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* Why does the project want to join CHIPS Alliance:
```
[the remark this answers is not in the document]
Comment (mithro) [reply projects/graduated/verible.md]: Done.

**Comment by mithro** [comment]
@mgielda - PTAL?

**Comment by mgielda** [comment]
Thanks! This is perfect. Vote ongoing.

**Comment by mgielda** [comment]
Vote passed! Congratulations :)

**Pull request merged**